portal/oauth/README.md

```diff
@@ -1,37 +1,39 @@
 # OAuth
 
-Use OAuth2 to authenticate with ArcGIS Online (or your own portal) to access secured resources.
+Authenticate with ArcGIS Online (or your own portal) using OAuth2 to access secured resources (such as private web maps or layers). Accessing secured items requires logging in to the portal that hosts them (an ArcGIS Online account, for example).
 
-![](OAuth.png)
+![Image of Authenticate with OAuth](OAuth.png)
```

Comment:
Our sample is only named `OAuth`, however I do think the longer name is better. We'll get to renaming the samples eventually!

```suggestion
![Image of OAuth](OAuth.png)
```

Reply:
from similar discussions with Android, renaming will come at a later date along side renaming the samples, so will keep it as it is for now for easier tracking

Reply:
What I meant was that the title of the readme is `OAuth`, so if we're keeping to the same format, the alt text should say `Image of OAuth` (as silly as that looks 😄 )